packages/adapters/gatewallet/src/utils.ts

```diff
@@ -0,0 +1,16 @@
+import { isInMobileBrowser } from '@tronweb3/tronwallet-abstract-adapter';
+
+export function supportGateWallet() {
+    return !!(window.gatewallet && window.gatewallet.tronLink);
+}
+
+export const isGateApp = /GateApp/i.test(navigator.userAgent);
+
+export function openGateWallet() {
+    if (!isGateApp && isInMobileBrowser()) {
+        window.location.href =
+            'https://gateio.onelink.me/DmA6/web3?dapp_url=' + encodeURIComponent(window.location.href);
```

Comment:
I could open gatewallet app with this deeplink but it just open the home page not the dapp browser. Is this a normal case ?

Reply:
had fixed